toolkit/smart-contracts/offchain/src/reserve/create.rs

```diff
@@ -137,3 +155,19 @@ fn reserve_validator_output(
 
 	amount_builder.with_minimum_ada_and_asset(&ma, ctx)?.build()
 }
+
+fn ics_validator_output(
+	scripts: &ReserveScripts,
+	ctx: &TransactionContext,
+) -> Result<TransactionOutput, JsError> {
+	let amount_builder = TransactionOutputBuilder::new()
+		.with_address(&scripts.illiquid_circulation_supply_validator.address(ctx.network))
+		.with_plutus_data(&TokenTransferDatumV1::ReserveTransfer.into())
```

Comment:
Why is this plutus data attachment required?
I can't see it being used in the validator smart-contract.